deploy/README.md

```diff
@@ -0,0 +1,112 @@
+Shaken Fist: Opinionated to the point of being impolite
```

Comment:
Should this be in the docs directory so it is served at http://shakenfist.com/?

Reply:
Yes. Especially that installation grid thingie. I'll do that now.

Reply:
Done.